### ISSUE TITLE
[EPIC] Accessibility Community of Practice - a11y Champs Program

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#90668]
## Description
The goal of the accessibility champions program is to provide self-paced learning resources and exercises to build knowledge around accessibility. 

Anyone working at the VA, with access to the DSVA/OCTO slack channel [#a11y-champs](https://dsva.slack.com/archives/C0552U2L30S), can become an accessibility champion [through the curriculum](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/vsa/accessibility/a11y-champs).

- The curriculum is a WIP.
- Ownership is a WIP. 
- This is a volunteer effort.

```[tasklist]
### a11y Champs Tasks
- [x] Complete Landing Page
- [x] Complete curriculum Level 1 - Community Member (see ticket [#57616](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57616))
- [ ] Determine program ownership and next steps 
- [ ] Complete curriculum Level 2 - Tester (see ticket [#57616](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57616))
- [ ] Complete curriculum Level 3 - Advocate (see ticket [#57616](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57616))
- [ ] Complete curriculum Level 4 - Apprentice (see ticket [#57616](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57616))
- [ ] Complete curriculum Level 5 - Champion  (see ticket [#57616](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57616))
```

```[tasklist]
### a11y Champs Tickets
- [ ] #65327 
- [ ] #57616 
```
